.github/workflows/release.yml

```diff
@@ -4,125 +4,141 @@ jobs:
-          iscc /dMyAppVersion="4.2.4" "${{ github.workspace }}/distribution/windows/cellprofiler.iss"
+          iscc /dMyAppVersion="${{ env.CP_VERSION }}" "${{ github.workspace }}/distribution/windows/cellprofiler.iss"
       - name: Mac - Create Artifact
-        if: ${{ matrix.os == 'macos-10.15' }}
+        if: startsWith(matrix.os, 'macos')
         uses: actions/upload-artifact@v1
         with:
-          name: CellProfiler-macOS-4.2.4.zip
-          path: ${{ github.workspace }}/distribution/macos/dist/CellProfiler-macOS-4.2.4.zip
+          name: CellProfiler-macOS-${{ env.CP_VERSION }}.zip
+          path: ${{ github.workspace }}/distribution/macos/dist/CellProfiler-macOS-${{ env.CP_VERSION }}.zip
       - name: Windows - Create Artifact
-        if: ${{ matrix.os == 'windows-2019' }}
+        if: startsWith(matrix.os, 'windows')
         uses: actions/upload-artifact@v2
         with:
-          name: CellProfiler-Windows-4.2.4.exe
-          path: ${{ github.workspace }}/distribution/windows/Output/CellProfiler-Windows-4.2.4.exe
+          name: CellProfiler-Windows-${{ env.CP_VERSION }}.exe
+          path: ${{ github.workspace }}/distribution/windows/Output/CellProfiler-Windows-${{ env.CP_VERSION }}.exe
 
-  upload-cp:
-    name: Upload CellProfiler
-    needs: build-cp
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Create Release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        id: create_release
-        uses: actions/create-release@v1
-        with:
-          draft: true
-          prerelease: true
-          release_name: ${{ github.ref }}
-          tag_name: ${{ github.ref }}
-      - name: Download Mac Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: CellProfiler-macOS-4.2.4.zip
-          path: ./
-      - name: Download Windows Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: CellProfiler-Windows-4.2.4.exe
-          path: ./
-      - name: Upload Mac Artifact
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        uses: actions/upload-release-asset@v1
-        with:
-          asset_content_type: application/zip
-          asset_name: CellProfiler-macOS-4.2.4.zip
-          asset_path: /home/runner/work/CellProfiler/CellProfiler/CellProfiler-macOS-4.2.4.zip
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-      - name: Upload Windows Artifact
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        uses: actions/upload-release-asset@v1
-        with:
-          asset_content_type: application/exe
-          asset_name: CellProfiler-Windows-4.2.4.exe
-          asset_path: /home/runner/work/CellProfiler/CellProfiler/CellProfiler-Windows-4.2.4.exe
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
+# TODO re-enable, make some condition for this to run
```

Comment:
This just makes the release accessible to try to download once GH has built it, it doesn't send it outside of GH. Should be safe to keep this enabled once you're done debugging without a separate condition - after this PR, hopefully will not be too many times where we want to test that it builds an executable, but we don't want to then take that executable and play with it/distribute it. Having auto upload to somewhere helpful like our AWS bucket is a good NEXT step, but that's not this bit.